.github/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest a feature idea for QGIS.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Wouldn't they all be? I don't see the value in this question.

Reply:
True, the default github template is a bit dumb. I simplified a lot.